src/transformers/models/big_bird/modeling_flax_big_bird.py

```diff
@@ -1048,15 +1075,23 @@ def _bigbird_block_rand_mask_with_head(
         # Total number of blocks in the mmask
         num_blocks = from_seq_length // from_block_size
         # Number of blocks per plan
-        plan_block_length = np.array(plan_from_length) // from_block_size
+        plan_block_length = jnp.array(plan_from_length) // from_block_size
         # till when to follow plan
         max_plan_idx = plan_from_length.index(from_seq_length)
+
         # Random Attention adjacency list
         rand_attn = [
-            np.zeros((num_blocks, np.sum(plan_num_rand_blocks[: max_plan_idx + 1])), dtype=np.int32)
+            jnp.zeros((num_blocks, sum(plan_num_rand_blocks[: max_plan_idx + 1])), dtype=jnp.int32)
```

Comment:
Would use a `jnp.sum` on jax arrays as it should be faster!
```suggestion
            jnp.zeros((num_blocks, jnp.sum(plan_num_rand_blocks[: max_plan_idx + 1])), dtype=jnp.int32)
```